distribution/knative/kustomization.yaml

```diff
@@ -2,36 +2,58 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 resources:
-- serving-core-v0_22_0.yaml
-- net-istio-v0_22_0.yaml
-- eventing-core-v0_22_0.yaml # Eventing is optional
+- serving-core-v0_24_0.yaml
+- net-istio-v0_24_0.yaml
+# - eventing-core-v0_22_0.yaml # Eventing is optional
+- https://github.com/kubernetes-sigs/metrics-server/releases/download/v0.5.0/components.yaml # metrics server (needed for HPA)
+- monitoring/
```

Comment:
Should probably disable monitoring here as well.

Reply:
We might need to think of a way to make it easier for people to switch the whole monitoring stack on/off